src/test/regress/expected/multi_partitioning_2.out

```diff
@@ -0,0 +1,1941 @@
+--
```

Comment:
I think we should not need this file. The diff between this and `multi_partitioning_1`.out is very, very small. I can see that few locks has changed which might be related to #2855 and its follow-ups.

I'm OK to remove this for now, and look the differences in detail later with a separate PR after we fix the lock issues.